src/SDKs/BotService/Management.BotService/generate.ps1

```diff
@@ -0,0 +1 @@
+﻿powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "botservice\resource-manager" -PowershellInvoker -AutoRestVersion "latest"
```

Comment:
Doesn't seem like the code is generated using the powershell script or the .cmd here. There should be a .txt log file and a sdkinfo*.cs file generated when you do so, please commit it,

Reply:
Oh I didn't know that part. Will do, thanks.

Reply:
@dsgouda when you ask to msbuild "build.proj", do you mean the build.proj file that is under azure-sdk-for-net\tools\legacy\ScriptBackup??? If I run your command on that file, I get this:

```
Project "E:\Repos\azure-sdk-for-net-ccastro\azure-sdk-for-net\tools\legacy\ScriptBackup\build.proj" on node 1 (build ta
rget(s)).
E:\Repos\azure-sdk-for-net-ccastro\azure-sdk-for-net\tools\legacy\ScriptBackup\build.proj(346,2): error MSB4019: The im
ported project "E:\Repos\azure-sdk-for-net-ccastro\azure-sdk-for-net\tools\legacy\ScriptBackup\tools\nuget.targets" was
 not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk.
Done Building Project "E:\Repos\azure-sdk-for-net-ccastro\azure-sdk-for-net\tools\legacy\ScriptBackup\build.proj" (buil
d target(s)) -- FAILED.
```

Reply:
Couple of things you can try
- Run `msbuild build.proj` which should download some build tools required
- Make sure you are running msbuild with a 2017 VS prompt

Reply:
Any ideas here will be appreciated :)

Reply:
There should be a `build.proj` in the root of the repo directory.

Reply:
Try setting `-ResourceProvider "botservice/resource-manager"`

Reply:
I have it, here is my ps1:

```
powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "botservice\resource-manager" -PowershellInvoker -AutoRestVersion "latest"
```

Reply:
It was the wrong slash in the ```-ResourceProvider "botservice/resource-manager"```, good catch 😀